fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -419,6 +420,8 @@ public ShowResultSet execute() throws AnalysisException, DdlException {
             handleShowFailPoint();
         } else if (stmt instanceof ShowDictionaryStmt) {
             handleShowDictionary();
+        } else if (stmt instanceof ShowBackendBlackListStmt) {
+            handleBackendBlackList();
```

Comment:
will this command pass to leader if served by follower?

Reply:
no, need execute on each FE